crates/cubecl-matmul/src/components/tile/plane_vec_mat_inner_product/setup.rs

```diff
@@ -31,6 +31,10 @@ where
         false
     }
 
+    fn can_cast_stage() -> bool {
```

Comment:
Can it be false if the compiler can't change the vectorization factor of a line?

Reply:
This is the confusion I was worried about but couldn't come up with a better name at the time 😅 
I think `can_cast_stage_elem` might be clearer. It's not about casting the stage itself, but the element after loading. There's no need to change vectorization for that.